### ISSUE TITLE
Accessibility issue: insufficient color contrast in score circles

### ISSUE THREAD

**Issue by kirillunlimited** [zachleat/speedlify-score#3]
## Description

Hi Zack 👋

I’m using this component on my site and noticed an accessibility issue. The score circles fail contrast checks in Lighthouse / PageSpeed Insights:

> Background and foreground colors do not have a sufficient contrast ratio

Since this component is often used on performance-focused sites, it’s a bit ironic that it can reduce Lighthouse scores due to contrast issues

Here are the screenshots to illustrate the issue.

<img width="480" alt="Image" src="https://github.com/user-attachments/assets/9fc76e54-9523-439f-96ed-c2c78cd3c931" />
<img width="480" alt="Image" src="https://github.com/user-attachments/assets/2df1d6c1-4a35-4190-9296-3eecf66f2aa3" />

## Possible solution

Right now, since the component uses Shadow DOM without styling hooks, it’s difficult to override styles externally. It would be great to expose color variables so that it could be possible to adjust these values manually to achieve the best contrast result

Thanks for the great project 🙌

**Comment by kirillunlimited** [comment]
I have just discovered that the color can be adjusted via `--spedlify-circle` variable. It solves the issue, but at the moment it is not possible to adjust the color of the text without changing the circle's border color, which would be nice to keep as it reflects the metric's status: good/ok/bad. I am closing the issue and opening a PR with the update: https://github.com/zachleat/speedlify-score/pull/4
